packages/dev/core/src/Materials/Textures/Loaders/ktxTextureLoader.ts

```diff
@@ -127,7 +127,7 @@ export class _KTXTextureLoader implements IInternalTextureLoader {
             );
         } else if (KhronosTextureContainer2.IsValid(data)) {
             const ktx2 = new KhronosTextureContainer2(texture.getEngine());
-            ktx2.uploadAsync(data, texture, options).then(
+            ktx2._uploadAsync(data, texture, options).then(

```

Comment:
Keeping an open comment to prevent merging for now. I will resolve this when ready